applications/acdc/src/acdc_agent_handler.erl

```diff
@@ -138,7 +138,7 @@ maybe_start_agent(AccountId, AgentId) ->
     case acdc_agents_sup:find_agent_supervisor(AccountId, AgentId) of
         'undefined' ->
             lager:debug("agent ~s (~s) not found, starting", [AgentId, AccountId]),
-            case kz_datamgr:open_doc(kz_util:format_account_id(AccountId, 'encoded'), AgentId) of
+            case kz_datamgr:open_doc(kzs_util:format_account_id(AccountId, 'encoded'), AgentId) of
```

Comment:
Can you run a sed command to convert all `kzs_util:format_account_id(.*, 'encoded')` to just `kzs_util:format_account_bd/1` please? Same for `'raw'`.